gui-apps/wf-recorder/wf-recorder-0.2.1.ebuild

```diff
@@ -7,20 +7,32 @@ inherit meson
 
 DESCRIPTION="Screen recorder for wlroots-based compositors"
 HOMEPAGE="https://github.com/ammen99/wf-recorder"
-SRC_URI="https://github.com/ammen99/${PN}/releases/download/v${PV}/${P}.tar.xz"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/ammen99/wf-recorder.git"
+else
+	SRC_URI="https://github.com/ammen99/wf-recorder/releases/download/v${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
```

Comment:
Where did the `arm64` come from? You can't add keywords.

Reply:
Oh, dang. All of @ammen99's packages are working on arm64 and I generally use them there too.  
I reflexively added it here, forgetting it wasn't present before. Will change it up.

Reply:
removed